playwright-dependencies/tests/smoke.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test('smokeTest', async ({ page }) => {
+  const containerPort = process.env.CONTAINER_PORT_EXPORTED;
+  const url = "localhost:" + containerPort;
+  
+  await page.goto(url);
+
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Email Address').click();
+  await page.getByLabel('Email Address').press('Home');
+  await page.getByLabel('Email Address').fill('test@liferay.com');
+  await page.getByLabel('Email Address').press('Tab');
+  await page.getByLabel('Password').fill('test');
+  await page.getByLabel('Password').press('Tab');
+  await page.getByLabel('Remember Me').press('Tab');
+  await page.getByLabel('Sign In- Loading').getByRole('button', { name: 'Sign In' }).press('Enter');
+  await page.waitForTimeout(3000);
+
+  await page.getByLabel('Password', { exact: true }).click();
+  await page.getByLabel('Password', { exact: true }).fill('a');
```

Comment:
@lucasmiranda0, can you please change to a more meaningful name, like `new-password`?

Reply:
Sure.